src/blog.html

```diff
@@ -0,0 +1,305 @@
+            </p>
+            <img class="testimonial-card__image" src="/image/testimony-2.png" alt="">
+            <h1 class="testimonial-card__name">Mark Mason</h1>
+            <h1 class="testimonial-card__position">Marketing Manager</h1>
+          </div>
+          <div class="testimonial-card">
+            <p class="testimonial-card__desc">
+              Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua.
+            </p>
+            <img class="testimonial-card__image" src="/image/testimony-3.png" alt="">
+            <h1 class="testimonial-card__name">Sam Preston</h1>
+            <h1 class="testimonial-card__position">CTO</h1>
+          </div></div>
+      </div>
+      <div class="cta">
+   <h1 class="title">Are you ready to grow your business with us?</h1>
+   <p class="cta__desc">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy.
+  </p>
+  <button class="btn">View Pricing</button>
+      </div>
+    </div>
+    <div class="footer-grad" style="position: absolute;
+    transform: rotate(180deg);
+    height: 254px;
+    right: 168px;
+    bottom: 300px;
+    
+    background: radial-gradient(483.9% 2719.65% at -49.5% -250%, #D798E1 17.55%, #9BFFA5 27.56%, #AED3FF 49.89%, #C9D4EF 56.53%, #CACFFA 65.69%) /* warning: gradient uses a rotation that is not supported by CSS and may not behave as expected */;
+    filter: blur(200px);">
+
+    </div>
+    <footer class="footer">
+    <div class="footer__upper">
+    <div class="footer__card">
+      <h1 class="footer__header">Company</h1>
+      <p class="footer__item">About Us</p>
+      <p class="footer__item">Why Choose us</p>
+      <p class="footer__item">Pricing</p>
+      <p class="footer__item">Testimonials</p>
+    </div>
+    <div class="footer__card">
+      <h1 class="footer__header">Resources</h1>
+      <p class="footer__item">Privacy Policy</p>
+      <p class="footer__item">Terms and Conditions</p>
+      <p class="footer__item">Blog</p>
+      <p class="footer__item">Contact Us</p>
+    </div>
+    <div class="footer__card">
+      <h1 class="footer__header">Products</h1>
+      <p class="footer__item">Project Management</p>
+      <p class="footer__item">Time tracker</p>
+      <p class="footer__item">Time Schedule</p>
+      <p class="footer__item">Lead generate</p>
+      <p class="footer__item">remote Collaboration</p>
+    </div>
+    <div class="footer__card">
+      <img class="footer__logo" src="/image/logo-2.png" alt="">
+      <p class="footer__subscribe">Subscribe to our Newsletter</p>
+      <div class="subscribe">
+        <input class="subscribe__input" type="text" placeholder="Enter Your Email">
+        <button class="btn btn--highlighted">Subscribe</button>
+      </div>
+    </div>
+    </div>
+    <div class="footer__lower">
```

Comment:
_footer__lower_ is not a good name.

what if you need to use it somewhere else?